hdf5.spec

```
@@ -1,29 +1,24 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %{!?_fmoddir:%global _fmoddir %{_libdir}/gfortran/modules}
 
-%global daos_major 0
-%global hdf5_commit fa40c6c59af5d9aabd4b478cd02f8a9f7ebf7922
-%define hdf5_sha .gfa40c6c59a
+%global daos_major 1
 
 # Patch version?
 %global snaprel %{nil}
-# HDF5 versions
-%global major 1.12
-%global minor 0
 
 # NOTE:  Try not to release new versions to released versions of Fedora
 # You need to recompile all users of HDF5 for each version change
 Name: hdf5
-Version: %{major}.%{minor}
-Release: 5%{hdf5_sha}%{?dist}
+Version: %{hdf5_major}_%{hdf5_minor}
```

Version components are separated by `.` not `_`:
```suggestion
Version: %{hdf5_major}.%{hdf5_minor}.%{hdf5_bugfix}%{?hdf5_prerelease:~%{hdf5_prerelease}}
```

This is going to build a package such as:

hdf5-1_13_0-...

which is not a properly formatted version and will not upgrade previous versions.  At least not in a/the way that it should.